### PR TITLE
Standardize on wget as downloader; fix a few other minor issues.

### LINKED ISSUE #458: Standardize on either wget or curl as dependency
Currently, both wget and curl are necessary to install GEMINI; the install script requires both to be present. The annotation-creation scripts certainly use a mix of both tools.

I think standardizing on either wget or curl would be good; I don't think it matters much which one.

I could prepare a patch to make the switch, if we can decide on one or the other.

What do you think?

### PR DESCRIPTION
**Pull request by cbrueffer**
This converts calls from curl to equivalent wget ones (standardizing on wget required fewer changes), and updates the documentation accordingly.

Also fixes a few more typos and a redundant dependency check.

Resolves #458

### CODE REVIEW

**Comment by brentp** [comment]
did you check all of these changes with a fresh install? I might hold off on this until next release as we already have a lot of moving parts.

**Comment by arq5x** [comment]
I agree that while this is a useful addition, it will keep things simpler if we add this into the next release.

**Comment by cbrueffer** [comment]
I didn't test a whole install, since the only curl -> wget change affecting installations is in ```install-data.py```, which I tested stand-alone. The rest of the functional changes is in the annotation_provenance scripts.

I'll do a full install over the weekend to confirm.

**Comment by cbrueffer** [comment]
I tested the installation with the patches proposed here, no issues.

**Comment by bgruening** [comment]
@cbrueffer can you rebase your branch. This is currently not ready to be merged.

**Comment by cbrueffer** [comment]
Done @bgruening, the changes still applied without conflicts.

**Comment by cbrueffer** [comment]
It doesn't look like this was merged. Not deemed useful?

**Comment by brentp** [comment]
Most of these changes are in internal script to create our annotation files. We have a lot of users that have issues with some of the dependencies used in gemini, but I have not heard curl or wget to be among them. 
I decided to close for that reason. If you feel strongly, you can re-open and I'll have a look at it once activity on master settles down.

**Comment by chapmanb** [comment]
+1 for including it. It always seemed confusing to be using both.

**Comment by cbrueffer** [comment]
Indeed. Also, the documentation implies that either curl or wget suffices, so I was a bit puzzled when I first tried installing Gemini and it complained about missing curl (the installer itself works with either, but the annotation downloader requires it).

**Comment by brentp** [comment]
@cbrueffer I'm repopening. could you do a full gemini install and test with this PR applied to master after adding `--continue` to the wget command?

I'd prefer to leave `annotation_provenance/*` unchanged as those are a record of how we created the annotation files.

**Comment by cbrueffer** [comment]
Thanks for opening the request again!

TL;DR: works fine

Gemini calls the ```install-data.py``` script which comes with the version it installs from conda, Because of that I tested this changeset by calling the patched ```gemini_install.py```, waiting until it calls the stock ```install-data.py``` (and fails because of missing curl), switching out the conda-installed version of ```install-data.py``` with the patched one and restarting the installation. Worked fine.
